toucan_connectors/google_big_query/google_big_query_connector.py

```diff
@@ -109,9 +131,49 @@ class Config:
         keep_untouched = (cached_property,)
 
     @staticmethod
-    def _get_google_credentials(credentials: GoogleCredentials, scopes: List[str]) -> Credentials:
-        credentials = get_google_oauth2_credentials(credentials).with_scopes(scopes)
-        return credentials
+    def _get_google_credentials(credentials: GoogleCredentials, scopes: list[str]) -> Credentials:
+        credentials = get_google_oauth2_credentials(credentials).with_scopes(scopes)  # type: ignore[assignment]
+        return credentials  # type: ignore[assignment]
```

Comment:
is there any reason ignoring the assignment warning instead of directly returning the value?

Reply:
a nit left with a battle with mypyand my code editor "lint" I guess, fixed !